assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -83,6 +83,35 @@ const SearchFunnelWidget = ( {
 		} )
 	);
 
+	// Get Analytics Goals data
```

Comment:
```suggestion
```

I don't think we need these comments, it's obvious from the select statements below what's happening 🙂 

Same goes for the comments below; I think you can remove them and it'll be fine.